extension-style/src/main/java/com/mapbox/maps/extension/style/atmosphere/generated/Atmosphere.kt

```diff
@@ -0,0 +1,496 @@
+   * Set the highColor property.
+   *
+   * The color of the atmosphere region above the horizon, `high-color` extends further above the horizon than the `color` property and its spread can be controlled with `horizon-blend`. The opacity can be set to `0` to remove the high atmosphere color contribution.
+   *
+   * @param highColor as String
+   */
+  fun highColor(highColor: String = "#245cdf"): Atmosphere
+
+  /**
+   * Set the highColor property
+   *
+   * @param highColor value of highColor as Expression
+   */
+  fun highColor(highColor: Expression): Atmosphere
+  /**
+   * Set the horizonBlend property.
+   *
+   * Horizon blend applies a smooth fade from the color of the atmosphere to the color of space. A value of zero leaves a sharp transition from atmosphere to space. Increasing the value blends the color of atmosphere into increasingly high angles of the sky.
+   *
+   * @param horizonBlend as Double
+   */
+  fun horizonBlend(horizonBlend: Double): Atmosphere
+
+  /**
+   * Set the horizonBlend property
+   *
+   * @param horizonBlend value of horizonBlend as Expression
+   */
+  fun horizonBlend(horizonBlend: Expression = Expression.fromRaw("""["interpolate",["linear"],["zoom"],4,0.2,7,0.1]""".trimIndent())): Atmosphere
+  /**
+   * Set the range property.
+   *
+   * The start and end distance range in which fog fades from fully transparent to fully opaque. The distance to the point at the center of the map is defined as zero, so that negative range values are closer to the camera, and positive values are farther away.
+   *
+   * @param range as List<Double>
+   */
+  fun range(range: List<Double> = listOf(0.5, 10.0)): Atmosphere
+
+  /**
+   * Set the range property
+   *
+   * @param range value of range as Expression
+   */
+  fun range(range: Expression): Atmosphere
+  /**
+   * Set the spaceColor property.
+   *
+   * The color of the region above the horizon and after the end of the `horizon-blend` contribution. The opacity can be set to `0` to have a transparent background.
+   *
+   * @param spaceColor as String
+   */
+  fun spaceColor(spaceColor: String): Atmosphere
+
+  /**
+   * Set the spaceColor property
+   *
+   * @param spaceColor value of spaceColor as Expression
+   */
+  fun spaceColor(spaceColor: Expression = Expression.fromRaw("""["interpolate",["linear"],["zoom"],4,"#010b19",7,"#367ab9"]""".trimIndent())): Atmosphere
+  /**
+   * Set the starIntensity property.
+   *
+   * A value controlling the star intensity where `0` will show no stars and `1` will show stars at their maximum intensity.
+   *
+   * @param starIntensity as Double
+   */
+  fun starIntensity(starIntensity: Double): Atmosphere
+
+  /**
+   * Set the starIntensity property
+   *
+   * @param starIntensity value of starIntensity as Expression
+   */
+  fun starIntensity(starIntensity: Expression = Expression.fromRaw("""["interpolate",["linear"],["zoom"],5,0.35,6,0]""".trimIndent())): Atmosphere
+}
+
+/**
+ * DSL function for [Atmosphere].
```

Comment:
```suggestion
 * Dsl function for [Atmosphere].
```

Reply:
We use capital letter everywhere else in generated files.